social/app/models/post.py

```diff
@@ -253,44 +252,96 @@ def get_remote_node_posts():
                 if len(some_json) == 0:
                     continue
 
-                for post_json in some_json['posts']:
```

Comment:
Why is the indentation of this block of code indented back one so that it's out of scope of the above for loop?